File: crates/harn-hostlib/tests/fixtures/ast/elixir/source.ex
defmodule Fixture.Greeter do
  def greet(name) do
    "hello #{name}"
  end

  defp internal_helper(s) do
    s
  end
end

defmodule Fixture.Shout do
  def shout(s), do: String.upcase(s)
end

File: crates/harn-hostlib/tests/fixtures/ast/ruby/source.rb
module Fixture
  class Greeter
    def initialize(name)
      @name = name
    end

    def greet
      "hello #{@name}"
    end

    def self.default
      new("world")
    end
  end

  def self.shout(s)
    s.upcase
  end
end
